app/src/main/java/at/bitfire/icsdroid/model/SubscriptionsModel.kt

```diff
@@ -40,10 +40,14 @@ import org.json.JSONException
 import java.io.FileInputStream
 import java.io.FileOutputStream
 import java.io.IOException
+import javax.inject.Inject
 
-class SubscriptionsModel(application: Application): AndroidViewModel(application) {
+class SubscriptionsModel @Inject constructor(
+    @ApplicationContext private val context: Context,
```

Comment:
Same case as above, shouldn't we continue using `AndroidViewModel`? Even if we inject the context from Hilt, it then gets passed to `AndroidViewModel`'s constructor.

Reply:
See https://github.com/bitfireAT/icsx5/pull/603#discussion_r2166073716